ansible/roles/xl_auth/vars/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 
+xl_auth_server_name: "{{ (inventory_hostname == 'default') | ternary(
```

Comment:
@mxtthias Does not work in production, as there's no "eth1" interface. Fix in #192